backend/infrahub/git/integrator.py

```diff
@@ -127,6 +129,7 @@ class InfrahubRepositoryIntegrator(InfrahubRepositoryBase):  # pylint: disable=t
     class that uses an "InfrahubRepository" or "InfrahubReadOnlyRepository" as input
     """
 
+    @flow(name="import-object-from-file", flow_run_name="Import objects")
```

Comment:
How about adding a task to add the repository as a related node using the "add_tags" function?